src/components/TimelineItemList.svelte

```diff
@@ -394,9 +397,9 @@
     flex: 1;
   }
 
-  .controls-header .st-button {
+  .filter-buttons {
```

Comment:
Positioning of the dropdown is now a little off?
<img width="550" alt="image" src="https://github.com/user-attachments/assets/8b978e70-5a14-4f86-bd25-3d42dbe2f375">